Scripts/Python/create_static_group.py

```diff
@@ -39,51 +39,39 @@
 from argparse import RawTextHelpFormatter
 import urllib3
 import requests
+from utils import authenticate_with_ome
```

Comment:
Undo and related code...So that customers can run the whole script independently.

Reply:
Done